src/main.ts

```diff
@@ -23,16 +25,28 @@ async function bootstrap() {
   const appConfig = configService.get<AppConfig>('appConfig');
   const mediaConfig = configService.get<MediaConfig>('mediaConfig');
 
-  const swaggerOptions = new DocumentBuilder()
-    .setTitle('HedgeDoc')
+  const publicApiOptions = new DocumentBuilder()
+    .setTitle('HedgeDoc Public API')
     .setVersion('2.0-dev')
     .addSecurity('token', {
       type: 'http',
       scheme: 'bearer',
     })
     .build();
-  const document = SwaggerModule.createDocument(app, swaggerOptions);
-  SwaggerModule.setup('apidoc', app, document);
+  const publicApi = SwaggerModule.createDocument(app, publicApiOptions, {
+    include: [PublicApiModule],
+  });
+  SwaggerModule.setup('apidoc', app, publicApi);
+
+  const privateApiOptions = new DocumentBuilder()
+    .setTitle('HedgeDoc Private API')
+    .setVersion('2.0-dev')
+    .build();
+
+  const privateApi = SwaggerModule.createDocument(app, privateApiOptions, {
+    include: [PrivateApiModule],
+  });
+  SwaggerModule.setup('private/apidoc', app, privateApi);
```

Comment:
Do we want to publish the private API in this form? Don't we encourage people to use the private changing api and buy us more support tickets in the future?

Reply:
What about only publishing the private APIdocs when we run with `NODE_ENV=development`?
Having quick access to the docs might be helpful when developing our frontend.